src/Compilers/Core/Portable/Emit/NoPia/CommonEmbeddedParameter.cs

```diff
@@ -79,7 +79,7 @@ private ImmutableArray<TAttributeData> GetAttributes(TModuleCompilationState com
                 // The constructors might be missing (for example, in metadata case) and doing lookup
                 // will ensure that we report appropriate errors.
 
-                foreach (var attrData in GetCustomAttributesToEmit(compilationState))
+                foreach (var attrData in GetCustomAttributesToEmit(moduleBuilder))
                 {
                     if (IsTargetAttribute(attrData, AttributeDescription.ParamArrayAttribute))
```

Comment:
No. will skip it and add tests for that.

---
In reply to: [113545413](https://github.com/dotnet/roslyn/pull/18715#discussion_r113545413) [](ancestors = 113545413)